articles/provider-honeywell.md

```diff
@@ -50,7 +50,7 @@ Billing information:
 
 | Parameter Name | Type     | Required | Description |
 |----------------|----------|----------|-------------|
-| `count`   | int    | No | Number of experimental shots. Defaults to 1. |
+| `count`   | int    | No | Number of experimental shots. Defaults to 500. |
```

Comment:
We may need to add more context here. If you were to manually submit a job to honeywell and omit the `count` argument then it _would_ in fact default to `1`. However, all of our tools (SDKs, CLIs) always set the param to 500.

Reply:
Is there any (supported) way in which one can submit a job manually?

Does this also apply to IonQ? The default [there](https://docs.microsoft.com/en-us/azure/quantum/provider-ionq) is stated as 500, is that a "tooling" default or the actual provider default?

Reply:
Anybody can call our APIs directly, or use our autogenerated SDKs for other languages (JS/Java/C#...).

This is a provider default

Reply:
Thanks! I updated the article to distinguish between the provider and tooling defaults.